tracks/ruby/exercises/high-scores/mentoring.md

```diff
@@ -0,0 +1,71 @@
+New concepts: Array query methods (+ chaining methods), instantiating a class, getter method or `attr_reader`.
```

Comment:
I've actually been adding "this exercise focusses on x y and z" to most of my note submission, so I really like that you're adding that as well. In general I tend to reference to the official ruby documentation, or javascript/typescript and I would encourage you linking the word `Array`, `class` and `attr_reader` here as well.

It helps the mentor because then they are encouraged to use the "official" docs as opposed to random articles, and can serve as a reminder / quick link, reducing the work for the mentor.

Reply:
Hmmm. I feel strongly that this is not the right place for links. I'll add it to the evaluation for this experiment.

Reply:
I meant it in a similar way like this: https://github.com/exercism/website-copy/pull/605/files#diff-832d9576c0235e6c66cf08e2eb0e0bdfR2, but perhaps you are right 🗡

Reply:
Aha, yes, links like to the Modulo arithmetic are marvellous!